index.html

```diff
@@ -98,25 +102,37 @@
   <script>
     const symbols = [
       "©", "®", "™", "“", "”", "£", "…", "½", "¼", "∞", "é", "á", "à", "ç", "€",
-      "¥", "₩", "¢", "¤", "œ", "Œ", "æ", "Æ", "✔", "°", "²", "³", "‽"
+      "¥", "₩", "¢", "¤", "œ", "Œ", "æ", "Æ", "✔", "°", "²", "³", "‽", { display: "ZWSP", character: "\u{200B}" }
     ];
 
     document.addEventListener("DOMContentLoaded", () => {
       const parent = document.querySelector(".symbols");
       for (const symbol of symbols) {
         const elem = document.createElement("div");
         elem.classList = "symbol";
-        elem.textContent = symbol;
+
+        let display, character;
+
+        if (typeof symbol === "string") {
```

Comment:
Apparently this is the way this is done in javascript?? `symbol instanceof String` was not working,,,

Reply:
yep, using `typeof` like this would be correct